hi-ml-azure/src/health_azure/utils.py

```diff
@@ -1459,6 +1460,35 @@ def download_files_from_run_id(
     torch_barrier()
 
 
+def get_driver_log_file_text(run: Run, download_file: bool = True) -> Optional[str]:
+    """
+    Returns text stored in run log driver file.
+
+    :param run: Run object representing the current run.
+    :param download_file: Boolean flag that control if the log file should be downloaded from the run or not.
```

Comment:
Suggestion: remove "Boolean flag to control". I'd just say maybe "If True, download the log file from the run.", where True must be surrounded by _two_ backticks.

Reply:
Why two backticks?

Reply:
Left a reference in my comment below.